sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/EventProcessorClient.cs

```diff
@@ -386,26 +393,37 @@ public bool IsRunning
         ///   passed only once, either as part of the connection string or separately.
         /// </remarks>
         ///
-        public EventProcessorClient(PartitionManager partitionManager,
+        public EventProcessorClient(BlobContainerClient blobContainerClient,
                                     string consumerGroup,
                                     string connectionString,
                                     string eventHubName,
                                     EventProcessorClientOptions clientOptions)
         {
-            Argument.AssertNotNull(partitionManager, nameof(partitionManager));
+            // BlobContainerClient validation is done by the BlobsCheckpointStore constructor.
+
             Argument.AssertNotNullOrEmpty(consumerGroup, nameof(consumerGroup));
             Argument.AssertNotNullOrEmpty(connectionString, nameof(connectionString));
 
-            clientOptions = clientOptions?.Clone() ?? new EventProcessorClientOptions();
```

Comment:
Stopped cloning it because we are not directly storing the client options anymore. We may have a race condition, but that's something we would have even with the `Clone` method. I think it's safe to delete it we keep it like this.

Reply:
I'd prefer that we clone.   We want to ensure that the child options get cloned for connections/retries, and stick to the pattern from other clients.